rapidsai/integration
# remove pins on geospatial libraries, RAPIDS nightlies

Contributes to https://github.com/rapidsai/build-planning/issues/56

With https://github.com/rapidsai/cuspatial/pull/1441, it should be possible to revert some of the workarounds introduced in #719.

## Notes for Reviewers

### How to test this

if this is working, we should see the following in the conda solves:

* `fmt >=11.0.2`
* `spdlog >=1.14.1`

We won't see `numpy >=2` yet, because `cugraph` doesn't support it yet (https://github.com/rapidsai/cugraph/pull/4615).
